apps/console/src/features/users/components/parent-user-invite-response-list.tsx

```diff
@@ -0,0 +1,310 @@
+            },
+            {
+                allowToggleVisibility: false,
+                dataIndex: "status",
+                id: "status",
+                key: "status",
+                render: (item: ParentUserInviteResponseListItem): ReactNode => {
+                    return (
+                        <Header as="h6" data-componentid={ `${componentId}-status-item-heading` } >
+                            <Header.Content>
+                                <Label
+                                    data-componentid={ `${componentId}-bulk-label` }
+                                    content={ item.status }
+                                    size="small"
+                                    color={ item.status === ParentOrgUserInviteResultStatus.SUCCESS ? "green" : "red" }
+                                    className="group-label"
+                                />
+                            </Header.Content>
+                        </Header>
+                    );
+                },
+                title: t("console:manage.features.user.modals.bulkImportUserWizard.wizardSummary.tableHeaders.status")
+            },
+            {
+                allowToggleVisibility: false,
+                dataIndex: "message",
+                id: "message",
+                key: "message",
+                render: (item: ParentUserInviteResponseListItem): ReactNode => {
+                    return (
+                        <Header as="h6" data-componentid={ `${componentId}-status-item-heading` }>
+                            <Header.Content>
+                                { item.message }
+                            </Header.Content>
+                        </Header>
+                    );
+                },
+                title: t("console:manage.features.user.modals.bulkImportUserWizard.wizardSummary.tableHeaders.message")
+            }
+        ];
+
+        return defaultColumns;
+    };
+
+    /**
+     * Shows list placeholders.
+     *
+     * @returns Placeholders.
+     */
+    const showPlaceholders = (): ReactElement => {
+        return (
+            <EmptyPlaceholder
+                data-testid={ `${componentId}-empty-search-result` }
+                data-componentid={ `${componentId}-empty-search-result` }
+                image={ getEmptyPlaceholderIllustrations().emptySearch }
+                imageSize="tiny"
+                title={ t("console:manage.features.users.usersList.search.emptyResultPlaceholder.title") }
+                subtitle=""
+            />
+        );
+    };
+
+    return (
+        <Grid>
+            <Grid.Row columns={ 1 }>
+                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                    {
+                        isLoading || hasError
+                            ? null
+                            : responseSummary?.failedCount === 0
```

Comment:
Will it be okay to show the success msg, based only on failedCount?

if both failedCount and successCount both can be sent as 0 from the backend due to some error, we will still show the success msg, which wouldn't be good UX :)

Reply:
These failedCount and successCount are calculated from the front end. If there are errors from back end hasError should be true therefore this success alert wont be visible.